isis/src/base/apps/spiceinit/spiceinit.cpp

```diff
@@ -144,15 +144,15 @@ namespace Isis {
       Kernel lk, pck, targetSpk, fk, ik, sclk, spk, iak, dem, exk;
       QList< priority_queue<Kernel> > ck;
       lk        = baseKernels.leapSecond(lab);
-      pck       = baseKernels.targetAttitudeShape(lab);
+      pck       = ckKernels.targetAttitudeShape(lab);
```

Comment:
I don't see a problem with this for now, since all previous pcks didn't have any quality value specified. Moving forward, is there any time we would want to add the ability to specify something like `pckpredicted=true` ?

Reply:
I considered this, but the current use case is when we solve for both instrument pointing and body rotation in the bundle adjust which doesn't need that. Also, for the current use case not requiring users to enter a separate parameter seemed better because there's no way to check that they entered one but not both in the kerneldb.

Reply:
This seems like a good solution. Even if we needed to add "pcksmithed" in the future this would not get in the way.